spec/factories/following.rb

```diff
@@ -0,0 +1,11 @@
+require './lib/pivotal_tracker'
+
+FactoryGirl.define do
+  factory :following, class: PivotalTracker::Following do
+    id        1
```

Comment:
Should be a https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences